.github/workflows/stale_issues_and_pr_cleanup.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      DAYS_BEFORE_STALE: 180
+      DAYS_BEFORE_CLOSE: 14
+    steps:
+      - uses: actions/stale@v7
+        with:
+          days-before-stale: ${{ env.DAYS_BEFORE_STALE }}
+          days-before-close: ${{ env.DAYS_BEFORE_CLOSE }}
+          stale-issue-message: 'This issue has not had any activity in ${{ env.DAYS_BEFORE_ISSUE_STALE }} days.  Cypress evolves quickly and the reported behavior should be tested on the latest version of Cypress to verify the behavior is still occurring.  It will be closed in ${{ env.DAYS_BEFORE_CLOSE }} days if no updates are provided.'
+          stale-pr-message: 'This PR has not had any activity in ${{ env.DAYS_BEFORE_PR_STALE }} days.  If no activity is detected in the next ${{ env.DAYS_BEFORE_CLOSE }} days, this PR will be closed.'
```

Comment:
```suggestion
          stale-issue-message: 'This issue has not had any activity in ${{ env.DAYS_BEFORE_STALE }} days.  Cypress evolves quickly and the reported behavior should be tested on the latest version of Cypress to verify the behavior is still occurring.  It will be closed in ${{ env.DAYS_BEFORE_CLOSE }} days if no updates are provided.'
          stale-pr-message: 'This PR has not had any activity in ${{ env.DAYS_BEFORE_STALE }} days.  If no activity is detected in the next ${{ env.DAYS_BEFORE_CLOSE }} days, this PR will be closed.'
```